src/it/resources/com/google/checkstyle/test/chapter4formatting/rule462horizontalwhitespace/InputWhitespaceAfter.java

```diff
@@ -0,0 +1,21 @@
+package com.google.checkstyle.test.chapter4formatting.rule462horizontalwhitespace;
+
+public class InputWhitespaceAfter {
+    public void check1(int x,int y){ //warn
+        for(int a=1,b=2;a<5;a++,b--); // warn
```

Comment:
please go through regression report and add more test case, like https://shashwat70.github.io/diff2/Vavr/index.html#A45

There should test case for valid and invalid for each point from https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
Please create valid Input file with all case and invalid Input file with violations (ideally such Inputs should almost identical and diff only in missed spaces in invalid file).

Reply:
If you do not want to add all tokens and tests for them to completely cover style chapter, as in original issue only comas were reported. It is ok, but please create separate issue on items that we miss even after your fix.

Reply:
> please create separate issue on items that we miss even after your fix.

moved to issue #8122 
I'll update its issue description in a few moments.

Reply:
All tokens that you adding in this PR, please cover by valid and invalid test cases

Reply:
Done

Reply:
I've added an example to the new issue. Please approve it and add required tags so that I may try to work on it. @romani

Reply:
My pleasure :)